### ISSUE TITLE
Unnecessary error log output from intent donation

### ISSUE THREAD

**Issue by marcpalmer** [MontanaFlossCo/Flint#224]
Output on action dispatch usually shows:

```
Error • N/A | Activity 'Flint Bootstrapping' | Flint/Core | Donation error: nil
```
